_releases/2022-07-06-june-2022.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+category: releases
+title: June 2022 Release Notes
+filename-custom: june-2022
+last_modified_at: July 6, 2022
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements
+
+* **Support for Cross-Posted Sitemaps**: We now support the ability to host your sitemap on an external site and index with our system. Ensure that your robots.txt lists the external sitemap url, and that the URLs on the sitemap match the domain you want to be indexed, illustrated below:
+
+**Robots.txt File on Main Domain: https://www.example.gov/robots.txt**
+```
+sitemap: https://www.anothersite.gov/sitemap.xml
+```
+
+**Externally Hosted Sitemap: https://www.anothersite.gov/sitemap.xml**
+```
+<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<url>
+<loc>https://example.gov/my-page.html</loc>
+<lastmod>2022-07-06T00:00:00+00:00</lastmod>
+</url>
+</urlset>
+``` 
+* **Security Enhancements**: We've improved our ability to respond to external attacks to the system.
+* **ElasticSearch Upgrade**: We’ve continued progress on upgrading our ElasticSearch version across our system.
```

Comment:
The apostrophe here and the one above don't match.  Not sure why.  I recommend making them consistent.  There is a small potential for some future system to choke on the difference.  The second one is probably from copy-paste from MS Word I'm guessing.